python/lsst/sattle/app/main.py

```diff
@@ -46,22 +55,44 @@ def read_tle_from_url(url, tle_source):
                 params = {
                     "id": norad_id,
                     "id_type": 'catalog',
-                    "start_date_jd": 2460637,
-                    "end_date_jd": 2460639, }
+                    "start_date_jd": start_time-0.6,
+                    "end_date_jd": start_time+0.6, }
 
                 base_url = "https://dev.satchecker.cps.iau.noirlab.edu"
```

Comment:
Suggest passing this as a variable, as it will one day not have "dev" in the URL

Reply:
Work do be done in DM-50395.